# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1428306135 -7200
#      Mon Apr 06 09:42:15 2015 +0200
# Node ID 7cefb9e7bc9ff0263e32d2a65d60d882797eb989
# Parent  107130a0490cd3705e70b73c32fb30c6b6af729d
* translators: added Felipe for pt_BR translation

diff --git a/libgui/languages/translators b/libgui/languages/translators
--- a/libgui/languages/translators
+++ b/libgui/languages/translators
@@ -2,14 +2,14 @@
 # to be able to inform them about generic translation file changes.
 # The generic file that can be used as a template for new
 # translation files is en_US.ts
 be_BY Mihas Varantsou <meequz@gmail.com>
 de_DE Andreas Weber <andy.weber.aw@googlemail.com>
 es_ES Valentin Ortega-Clavero <arcanos3030@gmail.com>
 fr_FR David Bateman <dbateman@free.fr>, Catalin Codreanu <codreanu.catalin@gmail.com>
 it_IT Massimiliano Fasi <massimiliano.fasi@gmail.com>
-pt_BR Júlio Hoffimann Mendes <julio.hoffimann@gmail.com>
+pt_BR Júlio Hoffimann Mendes <julio.hoffimann@gmail.com>, Felipe G. Nievinski <fgnievinski@gmail.com>
 pt_PT Carnë Draug <carandraug@octave.org>
 ru_RU Andriy Shinkarchuck <adriano32.gnu@gmail.com>
 uk_UA Andriy Shinkarchuck <adriano32.gnu@gmail.com>
 nl_NL Sander van Rijn <svr003@gmail.com>
 zh_CN Jeff Bai <aosc@members.fsf.org>
